src/components/Score.js

```diff
@@ -4,16 +4,41 @@ import { browserHistory } from 'react-router';
 import ScoreInput from './ScoreInput.js';
 
 const Score = React.createClass({
-  submitScores() {
-    browserHistory.push('/d/' + this.props.params.sessionId);
+  getInitialState() {
+    return {
+      scores: {}
+    };
+  },
+  saveScore() {
+    // TODO: validation - pk2
+    const data = JSON.stringify({
+      sessionId: this.state.sessionId,
+      scopes: this.state.scores
+    });
+    fetch('http://localhost:3000/api/score', {
```

Comment:
I think we need to import it here? Or is there some other sorcery going on where we made it global with webpack?